chef/cookbooks/hyperv/recipes/config.rb

```diff
@@ -56,12 +56,17 @@
 end
 
 %w{ OpenStackService.exe mkisofs.exe mkisofs_license.txt qemu-img.exe intl.dll libglib-2.0-0.dll libssp-0.dll zlib1.dll }.each do |bin_file|
-  cookbook_file "#{node[:openstack][:bin]}\\#{bin_file}" do
+  cookbook_file "#{node[:openstack][:bin]}/#{bin_file}" do
     source bin_file
   end
 end
 
-template "#{node[:openstack][:config]}\\nova.conf" do
+service "nova-compute" do
+  service_name node[:service][:nova][:name]
+  action [:enable, :start]
+end
```

Comment:
@tboerger see three comments above. I don't think the current code works on the very first chef run.